src/Miunie.WindowsApp/Views/ImpersonationChatPage.xaml

```diff
@@ -35,14 +35,7 @@
                         <ListView.ItemTemplate>
                             <DataTemplate x:DataType="core:MessageView">
                                 <StackPanel Orientation="Horizontal" Padding="10">
-                                    <Ellipse
-                                        Height="45"
-                                        Width="45"
-                                        VerticalAlignment="Top">
-                                        <Ellipse.Fill>
-                                            <ImageBrush ImageSource="{Binding AuthorAvatarUrl}"/>
-                                        </Ellipse.Fill>
-                                    </Ellipse>
+                                    <PersonPicture ProfilePicture="{Binding AuthorAvatarUrl}" Height="45" Width="45" VerticalAlignment="Top" />
```

Comment:
Sorry to bother you a little 😅, but there should be a property in PersonPicture for the user's name. It uses it when no picture is found, since it would use the first letter of their name. ;) See if you can find it. 🤔